### DISCUSSION TITLE
Prisma Studio

### DISCUSSION THREAD

**Discussion by fhub-1** [prisma/prisma#19176] [Q&A]
Hi everyone, I'm having an issue with Prisma Studio. When I run 'npx prisma studio' locally, it works perfectly fine and I can see my tables and data. However, when I navigate to the browser (http://localhost:5555/), the page is empty and doesn't show anything. I've tried restarting the server and clearing the cache, but nothing seems to work. Any ideas on what could be causing this issue and how to fix it?

**Comment by ludralph** [comment]
Hi @fhub-1 👋 

Can you please clarify what you mean by it works perfectly fine locally? When you run `npx prisma studio`, it launches a view of your table in the browser.

**Reply by fhub-1** [reply]
cant see my table just white screen

**Reply by ludralph** [reply]
This is strange behavior. Can you please share the version of Prisma you are running and also the operating system you are using? Does this happen on all browsers?

**Reply by fhub-1** [reply]
the new version

**Comment by fhub-1** [comment]
```bash
{
  "name": "prisma",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "build": "next build",
    "dev": "next dev",
    "postinstall": "prisma generate",
    "lint": "next lint",
    "start": "next start"
  },
  "dependencies": {
    "@prisma/client": "^4.11.0",
    "@t3-oss/env-nextjs": "^0.2.1",
    "next": "^13.4.1",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "zod": "^3.21.4"
  },
  "devDependencies": {
    "@types/eslint": "^8.21.3",
    "@types/node": "^18.15.5",
    "@types/prettier": "^2.7.2",
    "@types/react": "^18.0.28",
    "@types/react-dom": "^18.0.11",
    "@typescript-eslint/eslint-plugin": "^5.56.0",
    "@typescript-eslint/parser": "^5.56.0",
    "autoprefixer": "^10.4.14",
    "eslint": "^8.36.0",
    "eslint-config-next": "^13.4.1",
    "postcss": "^8.4.21",
    "prettier": "^2.8.6",
    "prettier-plugin-tailwindcss": "^0.2.6",
    "prisma": "^4.11.0",
    "tailwindcss": "^3.3.0",
    "typescript": "^5.0.2"
  },
  "ct3aMetadata": {
    "initVersion": "7.13.0"
  }
}
```
my package json file still not working with new project

**Reply by ludralph** [reply]
Hi @fhub-1 👋 

Can you share a repository so i can reproduce this?
